README.md

```diff
@@ -2,16 +2,79 @@
 
 [![License](https://img.shields.io/badge/License-Apache-blue.svg)](LICENSE)
 
-## Description
+# Realm Dart Samples
 
-This repo contains Realm Flutter and Realm Dart SDK samples. 
+This repo contains Flutter and Dart samples that demonstate the usage of [Realm Flutter and Dart Standalone SDKs](https://www.mongodb.com/docs/realm/sdk/flutter/).
 
-The Realm Flutter and Realm Dart SDK are located at https://github.com/realm/realm-dart
+To build your own app with Realm, use the [realm](https://pub.dev/packages/realm) package for Flutter and the [realm_dart](https://pub.dev/packages/realm_dart) package for Dart applications.
 
+The Realm Flutter and Realm Dart SDK repository is located at https://github.com/realm/realm-dart
 
-## Usage 
-Check the README.md files of the samples for instructions how to run them.
+You may find the [Quick Start - Flutter SDK]( https://www.mongodb.com/docs/realm/sdk/flutter/quick-start/) helpful if you are running using Realm for the first time.
+You can also read more about using the Realm with [Atlas Device Sync](https://www.mongodb.com/docs/realm/sdk/flutter/sync/) and connecting to [Atlas App Services](https://www.mongodb.com/docs/realm/sdk/flutter/app-services/).
 
+## [`realm_dart`](https://github.com/realm/realm-dart-samples/tree/main/realm_dart) - Dart console application
 
+This sample is using [Realm Dart SDK](https://www.mongodb.com/docs/realm/sdk/flutter/#dart-standalone-realm) to demonstrates the following simple operations with a local realm:
 
-##### The "Dart" name and logo and the "Flutter" name and logo are trademarks owned by Google. 
+- open/create local realm
+- add/edit objects to realm
+- read all object
+- query objects from realm
+- close realm
+
+To run the app follow the instructions in the README: [realm_dart\README.md](https://github.com/realm/realm-dart-samples/tree/main/realm_dart#readme)
+
+## [`provider_shopper`](https://github.com/realm/realm-dart-samples/tree/main/provider_shopper) - Flutter application
+
+A Flutter sample app from [flutter sample repo](https://github.com/flutter/samples/tree/master/provider_shopper) that shows a state management approach. This app is modified to be integrated with a local realm using [Realm Flutter SDK](https://www.mongodb.com/docs/realm/sdk/flutter/).
+The list with all the available items are initialized into `CatalogModel` class as realm objects added to a realm.
+
+If the users add any items to their shopping cart, these irems are added to a memory collection in `CartModel` class. Only initial list of items is stored to the realm.
```

Comment:
i have trouble understanding these sentences can you clarify/expand upon what you mean?

Reply:
I will remove them. They are too specific and cover some implementation details.